### PR TITLE
Enhancing User Experience: Implementing Pagination for Streamlined Data Navigation

### LINKED ISSUE #49: Implement infinite scroll [using React Query maybe]

### PR DESCRIPTION
**Pull request by aimdexter**
## Key Changes and Enhancements:

1. Pagination with Atom Jotai Store: Integrated the Atom Jotai store into the project to manage the state and handle the pagination logic.

2. Data Slicing: To optimize performance and reduce the amount of data loaded at once, I have sliced the dataset into smaller chunks of 12 objects each. This ensures that only a manageable amount of data is displayed on each page.

3. Page Calculation: I have implemented a mechanism to calculate the total number of pages based on the size of the dataset and the number of objects per page. This dynamic calculation ensures that users always have an accurate overview of the available data pages.

4. Pagination Buttons: added pagination buttons, displaying page numbers and allowing users to navigate between different pages.

## Related Issues

Fixes #49

Please review and merge this PR. Thank you!

### CODE REVIEW

**Comment by soufyanelf** [comment]
The mobile display doesn't work well
image

**Comment by aimdexter** [comment]
Alright I'll double check it.

**Comment by aimdexter** [comment]
@soufyanelf Style for all breakpoints is fixed now.
image

File: src/app/helps/help-cards.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
+"use client";
```
Comment (soufyanelf) [inline src/app/helps/help-cards.jsx:1]: Can you please use camel case for file naming.

**Review by soufyanelf — APPROVED** [review]

**Comment by soufyanelf** [comment]
One last comment: Can you please use camel case for file naming.
I suggest helpCardsPage

**Comment by aimdexter** [comment]
> One last comment: Can you please use camel case for file naming. I suggest helpCardsPage

yes sure

**Pull request merged**